share/doc/homebrew/Formula-Cookbook.md

```diff
@@ -345,79 +275,60 @@ Thus, if you change the name of the class, you must also rename the file. Filena
 
-New formulae being submitted to Homebrew should run `brew audit <formula name> --strict --online`. This command is performed by the Brew Test Bot on new submissions as part of the automated build and test process, and highlights more potential issues than the standard audit.
+New formulae being submitted to Homebrew should run `brew audit --strict --online foo`. This command is performed by the Brew Test Bot on new submissions as part of the automated build and test process, and highlights more potential issues than the standard audit.
 
 Use `brew info` and check if the version guessed by Homebrew from the URL is
-correct. Add an explicit `version` if not.
+correct. Add an explicit [`version`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#version-class_method) if not.
 
 ## Commit
 
 Everything is built on Git, so contribution is easy:
 
-    brew install git # if you already have git installed, skip this command
-    brew update # required in more ways than you think (initializes the brew git repository if you don't already have it)
-    cd `brew --repository`
-    # Create a new git branch for your formula so your pull request is easy to
-    # modify if any changes come up during review.
-    git checkout -b <some-descriptive-name>
-    git add Library/Formula/foo.rb
-    git commit
+```shell
+brew update # required in more ways than you think (initializes the brew git repository if you don't already have it)
+cd /usr/local
+# Create a new git branch for your formula so your pull request is easy to
+# modify if any changes come up during review.
+git checkout -b <some-descriptive-name>
+git add Library/Formula/foo.rb
+git commit
+```
 
 The established standard for Git commit messages is:
 
-* the first line is a commit summary of *50 characters or less*, then
+* the first line is a commit summary of *50 characters or less*
 * two (2) newlines, then
 * explain the commit throughly
 
-At Homebrew, we like to put the name of the formula up front like so: "foobar 7.3 (new formula)".
+At Homebrew, we like to put the name of the formula up front like so: `foobar 7.3 (new formula)`.
 This may seem crazy short, but you’ll find that forcing yourself to summarise the commit encourages you to be atomic and concise. If you can’t summarise it in 50-80 characters, you’re probably trying to commit two commits as one. For a more thorough explanation, please read Tim Pope’s excellent blog post, [A Note About Git Commit Messages](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
 
-The preferred commit message format for simple version updates is "foobar 7.3".
-
-Ensure you reference any relevant GitHub issue `#12345` in the commit message. Homebrew’s history is the first thing future contributors will look to when trying to understand the current state of formulae they’re interested in.
+The preferred commit message format for simple version updates is `foobar 7.3` and for fixes is `foobar: fix flibble matrix.`.
```

Comment:
Judging from the `git log` on `Library/Homebrew` you're more or less the only maintainer to end commit subjects with a full stop. We don't have to regulate commit messages to that level of detail, but maybe it is relevant to how we choose our examples?